mfr/extensions/codepygments/static/css/default.css

```diff
@@ -81,11 +82,12 @@ code, kbd, pre, samp {
     box-sizing: border-box;
 }
 
-/* CSS only for Firefox */
+/* CSS only for Firefox
 @-moz-document url-prefix() {
   .highlight > pre {
-      white-space: pre-wrap;       /*  css-3 */
-      white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
+      white-space: pre-wrap;
+      white-space: -moz-pre-wrap;
       word-break: break-all;
   }
 }
+*/
```

Comment:
Instead of commenting this out, please delete it.  We'll still have it in the git history.